thirdparty/README.md

```diff
@@ -1,23 +1,14 @@
 # Third party libraries
 
+## fbx parser
```

Comment:
Thanks for thinking of adding relevant info here. This file is specifically matching the tree contents of `thirdparty/`, so I think it's best to document the provenance of `modules/fbx/fbx_parser` directly in `modules/fbx/readme.md` or in a new readme in `modules/fbx/fbx_parser/`.

Reply:
done!